Tools/AP_Periph/README.md

```diff
@@ -29,7 +29,6 @@ UAVCAN sensor types. Support is included for:
  - Barometers (SPI or I2C)
  - Airspeed sensors (I2C)
  - Rangefinders (UART or I2C)
- - ADSB (Ping ADSB receiver on UART)
```

Comment:
This should be `uAvionix Ping ADSB`... 

The line can be here, but the current version is misleading.

I'll be working on getting proper support for the library in very soon.

Reply:
then what was this about? https://discuss.ardupilot.org/t/adsb-using-ap-periph-not-connecting/105634/4

Reply:
... at the risk of being pushy, maybe give @peterbarker a few days and if it remains a WIP then we remove it for now and put it back when his PR is ready... or just remove it now and put it back later.

Reply:
> then what was this about? https://discuss.ardupilot.org/t/adsb-using-ap-periph-not-connecting/105634/4

I've responded on that Discuss post.